sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/ClientCreationWithAzureActiveDirectoryCredentialSample.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai;
+
+import com.azure.ai.openai.models.Completions;
+import com.azure.ai.openai.models.CompletionsOptions;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Sample demonstrates how to create a client with `AzureKeyCredential`.
+ */
+public class ClientCreationWithAzureActiveDirectoryCredentialSample {
```

Comment:
Just in the spirit of the sample work that Brandon is doing, would you oppose to moving these 3 samples to a subpackage called `auth`? Then we really tie them together conceptually and users can find easily the alternatives to authenticate their clients.

Reply:
Yeah. will do